<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(54, 54, 54);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="mainTab">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>40</y>
      <width>1211</width>
      <height>741</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
color: rgb(96, 96, 96);
font: 16pt &quot;Verdana&quot;;


QTabBar::tab {
 background-color: rgb(96, 96, 96);
 color: rgb(96, 96, 96);

}
</string>
    </property>
    <property name="tabShape">
     <enum>QTabWidget::Rounded</enum>
    </property>
    <property name="currentIndex">
     <number>3</number>
    </property>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>Maintenance</string>
     </attribute>
     <widget class="QFrame" name="frame_2">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>-30</y>
        <width>21</width>
        <height>791</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">
background-color: rgb(181, 181, 181);</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
     <widget class="QWidget" name="gridLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>270</y>
        <width>191</width>
        <height>201</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <property name="spacing">
        <number>0</number>
       </property>
       <item row="3" column="0">
        <widget class="QPushButton" name="deleteTaskButton">
         <property name="maximumSize">
          <size>
           <width>120</width>
           <height>60</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">
QPushButton{
color: rgb(54, 54, 54);
border-color: rgb(96, 96, 96);
selection-color: rgb(0, 0, 0);
font: 10pt &quot;Verdana&quot;;
border-radius: 10px;
background-color: rgb(181, 181, 181);

}

QPushButton:hover {
color: rgb(255, 159, 102);
}</string>
         </property>
         <property name="text">
          <string>Delete task</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QPushButton" name="addTaskButton">
         <property name="maximumSize">
          <size>
           <width>120</width>
           <height>60</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">

QPushButton{
color: rgb(54, 54, 54);
border-color: rgb(96, 96, 96);
selection-color: rgb(0, 0, 0);
font: 10pt &quot;Verdana&quot;;
border-radius: 10px;
background-color: rgb(181, 181, 181);

}

QPushButton:hover {
color: rgb(255, 159, 102);
}</string>
         </property>
         <property name="text">
          <string>Add task</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QPushButton" name="editTaskButton">
         <property name="maximumSize">
          <size>
           <width>120</width>
           <height>60</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">
QPushButton{
color: rgb(54, 54, 54);
border-color: rgb(96, 96, 96);
selection-color: rgb(0, 0, 0);
font: 10pt &quot;Verdana&quot;;
border-radius: 10px;
background-color: rgb(181, 181, 181);

}

QPushButton:hover {
color: rgb(255, 159, 102);
}</string>
         </property>
         <property name="text">
          <string>Edit task</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QFrame" name="frame_5">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>210</y>
        <width>631</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius: 10px;
background-color: rgb(181, 181, 181);</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLabel" name="logoLabel2">
       <property name="geometry">
        <rect>
         <x>620</x>
         <y>-280</y>
         <width>261</width>
         <height>321</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="LOGO2.qrc">:/LogoInside/LOGO INSIDE.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>10</y>
        <width>261</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">
border-color: rgb(96, 96, 96);
selection-color: rgb(0, 0, 0);

background-color: rgb(96, 96, 96);

color: rgb(255, 255, 255); 
font: 16pt &quot;Verdana&quot;;</string>
      </property>
      <property name="text">
       <string>Current Tasks</string>
      </property>
     </widget>
     <widget class="QTableWidget" name="plannedTasksTable">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>270</y>
        <width>771</width>
        <height>401</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 12pt &quot;Verdana&quot;;
color: rgb(0, 0, 0);
background-color: rgb(255, 255, 255);</string>
      </property>
      <column>
       <property name="text">
        <string>Task ID</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Type</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Machine ID</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Description</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Created</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>End </string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Part</string>
       </property>
      </column>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>240</y>
        <width>231</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">
border-color: rgb(96, 96, 96);
selection-color: rgb(0, 0, 0);
background-color: rgb(96, 96, 96);
color: rgb(255, 255, 255); 
font: 16pt &quot;Verdana&quot;;</string>
      </property>
      <property name="text">
       <string>Planned tasks</string>
      </property>
     </widget>
     <widget class="QPushButton" name="taskDoneButton">
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>10</y>
        <width>111</width>
        <height>21</height>
       </rect>
      </property>
      <property name="maximumSize">
       <size>
        <width>120</width>
        <height>60</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">

QPushButton{
color: rgb(54, 54, 54);
border-color: rgb(96, 96, 96);
selection-color: rgb(0, 0, 0);
font: 10pt &quot;Verdana&quot;;
border-radius: 10px;
background-color: rgb(181, 181, 181);

}

QPushButton:hover {
color: rgb(255, 159, 102);
}</string>
      </property>
      <property name="text">
       <string>Task Done</string>
      </property>
     </widget>
     <widget class="QTableWidget" name="currentTasksTable">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>40</y>
        <width>771</width>
        <height>161</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 12pt &quot;Verdana&quot;;
color: rgb(0, 0, 0);
background-color: rgb(255, 255, 255);</string>
      </property>
      <column>
       <property name="text">
        <string>Task ID</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Type</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Machine ID</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Description</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Created</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>End </string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Part</string>
       </property>
      </column>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Machines</string>
     </attribute>
     <widget class="QFrame" name="frame_3">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>-40</y>
        <width>21</width>
        <height>781</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">
background-color: rgb(181, 181, 181);</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
     <widget class="QTableWidget" name="machineTable">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>10</y>
        <width>791</width>
        <height>661</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 12pt &quot;Verdana&quot;;
color: rgb(0, 0, 0);
background-color: rgb(255, 255, 255);</string>
      </property>
      <column>
       <property name="text">
        <string>Machine ID</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Model</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Required Maintenance</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Is Running</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Operator</string>
       </property>
      </column>
     </widget>
     <widget class="QWidget" name="gridLayoutWidget_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>191</width>
        <height>201</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_4">
       <property name="spacing">
        <number>0</number>
       </property>
       <item row="4" column="0">
        <widget class="QPushButton" name="editMachineButton">
         <property name="maximumSize">
          <size>
           <width>120</width>
           <height>60</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">
QPushButton{
color: rgb(54, 54, 54);
border-color: rgb(96, 96, 96);
selection-color: rgb(0, 0, 0);
font: 10pt &quot;Verdana&quot;;
border-radius: 10px;
background-color: rgb(181, 181, 181);

}

QPushButton:hover {
color: rgb(255, 159, 102);
}</string>
         </property>
         <property name="text">
          <string>Edit machine</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QPushButton" name="deleteMachineButton">
         <property name="maximumSize">
          <size>
           <width>120</width>
           <height>60</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">
QPushButton{
color: rgb(54, 54, 54);
border-color: rgb(96, 96, 96);
selection-color: rgb(0, 0, 0);
font: 10pt &quot;Verdana&quot;;
border-radius: 10px;
background-color: rgb(181, 181, 181);

}

QPushButton:hover {
color: rgb(255, 159, 102);
}</string>
         </property>
         <property name="text">
          <string>Delete machine</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QPushButton" name="addMachineButton">
         <property name="maximumSize">
          <size>
           <width>120</width>
           <height>60</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">

QPushButton{
color: rgb(54, 54, 54);
border-color: rgb(96, 96, 96);
selection-color: rgb(0, 0, 0);
font: 10pt &quot;Verdana&quot;;
border-radius: 10px;
background-color: rgb(181, 181, 181);

}

QPushButton:hover {
color: rgb(255, 159, 102);
}</string>
         </property>
         <property name="text">
          <string>Add machine</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Employees</string>
     </attribute>
     <widget class="QFrame" name="frame">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>-40</y>
        <width>21</width>
        <height>761</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">
background-color: rgb(181, 181, 181);</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
     <widget class="QWidget" name="gridLayoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>191</width>
        <height>201</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <item row="4" column="0">
        <widget class="QPushButton" name="editEmployeeButton">
         <property name="maximumSize">
          <size>
           <width>120</width>
           <height>60</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">
QPushButton{
color: rgb(54, 54, 54);
border-color: rgb(96, 96, 96);
selection-color: rgb(0, 0, 0);
font: 10pt &quot;Verdana&quot;;
border-radius: 10px;
background-color: rgb(181, 181, 181);

}

QPushButton:hover {
color: rgb(255, 159, 102);
}</string>
         </property>
         <property name="text">
          <string>Edit employee</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QPushButton" name="deleteEmployeeButton">
         <property name="maximumSize">
          <size>
           <width>120</width>
           <height>60</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">
QPushButton{
color: rgb(54, 54, 54);
border-color: rgb(96, 96, 96);
selection-color: rgb(0, 0, 0);
font: 10pt &quot;Verdana&quot;;
border-radius: 10px;
background-color: rgb(181, 181, 181);

}

QPushButton:hover {
color: rgb(255, 159, 102);
}</string>
         </property>
         <property name="text">
          <string>Delete employee</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QPushButton" name="addEmployeeButton">
         <property name="maximumSize">
          <size>
           <width>120</width>
           <height>60</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">

QPushButton{
color: rgb(54, 54, 54);
border-color: rgb(96, 96, 96);
selection-color: rgb(0, 0, 0);
font: 10pt &quot;Verdana&quot;;
border-radius: 10px;
background-color: rgb(181, 181, 181);

}

QPushButton:hover {
color: rgb(255, 159, 102);
}</string>
         </property>
         <property name="text">
          <string>Add employee</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QTableWidget" name="employeesTable">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>10</y>
        <width>791</width>
        <height>661</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 12pt &quot;Verdana&quot;;
color: rgb(0, 0, 0);

background-color: rgb(255, 255, 255);</string>
      </property>
      <column>
       <property name="text">
        <string>Worker ID</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Name</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Surname</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Operator</string>
       </property>
      </column>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Parts</string>
     </attribute>
     <widget class="QFrame" name="frame_4">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>-30</y>
        <width>21</width>
        <height>761</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">
background-color: rgb(181, 181, 181);</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
     <widget class="QWidget" name="gridLayoutWidget_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>191</width>
        <height>201</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <property name="spacing">
        <number>0</number>
       </property>
       <item row="3" column="0">
        <widget class="QPushButton" name="deletePartButton">
         <property name="maximumSize">
          <size>
           <width>120</width>
           <height>60</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">
QPushButton{
color: rgb(54, 54, 54);
border-color: rgb(96, 96, 96);
selection-color: rgb(0, 0, 0);
font: 10pt &quot;Verdana&quot;;
border-radius: 10px;
background-color: rgb(181, 181, 181);

}

QPushButton:hover {
color: rgb(255, 159, 102);
}</string>
         </property>
         <property name="text">
          <string>Delete part</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QPushButton" name="editPartButton">
         <property name="maximumSize">
          <size>
           <width>120</width>
           <height>60</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">
QPushButton{
color: rgb(54, 54, 54);
border-color: rgb(96, 96, 96);
selection-color: rgb(0, 0, 0);
font: 10pt &quot;Verdana&quot;;
border-radius: 10px;
background-color: rgb(181, 181, 181);

}

QPushButton:hover {
color: rgb(255, 159, 102);
}</string>
         </property>
         <property name="text">
          <string>Edit part</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QPushButton" name="addPartButton">
         <property name="maximumSize">
          <size>
           <width>120</width>
           <height>60</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">

QPushButton{
color: rgb(54, 54, 54);
border-color: rgb(96, 96, 96);
selection-color: rgb(0, 0, 0);
font: 10pt &quot;Verdana&quot;;
border-radius: 10px;
background-color: rgb(181, 181, 181);

}

QPushButton:hover {
color: rgb(255, 159, 102);
}</string>
         </property>
         <property name="text">
          <string>Add part</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QTableWidget" name="partsTable">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>10</y>
        <width>600</width>
        <height>661</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 12pt &quot;Verdana&quot;;
color: rgb(0, 0, 0);
background-color: rgb(255, 255, 255);</string>
      </property>
      <column>
       <property name="text">
        <string>Name</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Price</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Stock</string>
       </property>
      </column>
     </widget>
     <widget class="QLineEdit" name="partsNameSearch">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>340</y>
        <width>189</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="partsNameSearchButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>380</y>
        <width>120</width>
        <height>31</height>
       </rect>
      </property>
      <property name="maximumSize">
       <size>
        <width>120</width>
        <height>60</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">
QPushButton{
color: rgb(54, 54, 54);
border-color: rgb(96, 96, 96);
selection-color: rgb(0, 0, 0);
font: 10pt &quot;Verdana&quot;;
border-radius: 10px;
background-color: rgb(181, 181, 181);

}

QPushButton:hover {
color: rgb(255, 159, 102);
}</string>
      </property>
      <property name="text">
       <string>Search by name</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="dateTimeLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>431</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255); 
font: 16pt &quot;Verdana&quot;;</string>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="insideLabel">
    <property name="geometry">
     <rect>
      <x>1080</x>
      <y>-20</y>
      <width>121</width>
      <height>121</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="LOGO2.qrc">:/LogoInside/LOGO INSIDE.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <zorder>insideLabel</zorder>
   <zorder>mainTab</zorder>
   <zorder>dateTimeLabel</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="LOGO2.qrc"/>
 </resources>
 <connections/>
</ui>
